tests/kernel/mem_heap/k_heap_api/src/test_kheap_api.c

```diff
@@ -105,9 +119,38 @@ void test_k_heap_free(void)
 /**
  * @brief Validate allocation and free heap memory in isr context.
  *
+ * @details The test validates k_heap_alloc() in isr context, the timeout
+ * para should be K_NO_WAIT, because this situation isn't allow to wait.
```

Comment:
is para a typo or intentional?

Reply:
Thank you for you careful review. I'll changed this word to param.